### PR TITLE
Disable chrony daemon from acting as server.

### PR DESCRIPTION
**Pull request by adelton**
#### Description:

- Disable chrony daemon from acting as server.

#### Rationale:

- Minimizing the exposure of the server functionality of the chrony daemon diminishes the attack space.

### CODE REVIEW

**Comment by adelton** [comment]
@stevegrubb, could you please provide `identifiers`, `references`, and into what section in `ospp.profile` the rule should fall?

**Comment by stevegrubb** [comment]
> @stevegrubb, could you please provide `identifiers`, `references`, and into what section in `ospp.profile` the rule should fall?

This chrony rule falls under FMT_SMF_EXT.1 for its reference. Identifiers are issued by Shawn. He should have some numbers that can be assigned. As for what section, it should be under the services section.

**Comment by shawndwells** [comment]
> > @stevegrubb, could you please provide `identifiers`, `references`, and into what section in `ospp.profile` the rule should fall?
> 
> This chrony rule falls under FMT_SMF_EXT.1 for its reference. Identifiers are issued by Shawn. He should have some numbers that can be assigned. As for what section, it should be under the services section.

Grab any available CCE from the available list. Suggest picking one from the middle-ish of the list to avoid creating merge conflicts with existing PRs.

**Comment by jan-cerny** [comment]
@adelton The list of available CCEs is in https://github.com/ComplianceAsCode/content/blob/master/shared/references/cce-redhat-avail.txt In your patch, you can remove one number from this file and add the  number to `cce@rhel8:` in `rule.yml`.

**Comment by adelton** [comment]
Thanks you all. Updated per recommendation, rebased on master -> d57dd050792c6fb4461796c2f0898a5502984911. @stevegrubb, not sure if I managed to put it exactly where you envisioned it to be.

**Comment by adelton** [comment]
I could use `replace_or_append` in the remediation script (similar to https://github.com/ComplianceAsCode/content/pull/4449) if that's viewed as the preferred approach.

**Comment by yuumasato** [comment]
@adelton I think we can stick with `replace_or_append` for now, it will be reworked by @cipherboy, and this way it can be easily tracked.

File: linux_os/guide/services/ntp/chronyd_client_only/rule.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+title: 'Disable chrony daemon from acting as server'
+
+description: |-
+    The <tt>port</tt> option in <tt>/etc/chrony.conf</tt> can be set to
+    <tt>0</tt> to make chrony daemon to never open any listening port
+    for server operation and to operate strictly in a client-only mode.
+
+rationale: |-
+    Minimizing the exposure of the server functionality of the chrony
+    daemon diminishes the attack space.
```
Comment (yuumasato) [inline linux_os/guide/services/ntp/chronyd_client_only/rule.yml]: nitpick: I think a more broadly used term is `attack surface`.

Comment (adelton) [reply linux_os/guide/services/ntp/chronyd_client_only/rule.yml]: Fixed in 430c2d727d117cacff5ff0ebd50df78cc3671251.

**Comment by adelton** [comment]
> we can stick with `replace_or_append`

Done in 430c2d727d117cacff5ff0ebd50df78cc3671251. Also rebased on master.

**Comment by jan-cerny** [comment]
@adelton please resolve the conflicts

**Comment by adelton** [comment]
Rebased on master -> 3e9ddff1050d29a7b4d84046fb298d4827d58429.

**Comment by jan-cerny** [comment]
Thank you.

**Pull request merged**